### DISCUSSION TITLE
Mapping Mac Keyboard to Windows

### DISCUSSION THREAD

**Discussion by w3us** [rustdesk/rustdesk#3079] [Q&A]
I am running rustdesk on a MacBookPro connecting remotely to a Windows 10 PC.  The keyboard doesn't map directly for some of the <CTRL> and function key functions.  Anydesk offers a "translate" keyboard option.  If rustdesk has such an option, I've been unable o find it. Help please.

**Comment by rustdesk** [comment]
https://github.com/rustdesk/rustdesk/issues/2365

**Comment by soxft** [comment]
The same question.
I hope that trustdesk can support key mapping settings, such as `control` mapping to `command`, `command` mapping to `ctrl`, and users can also customize these settings.
Todesk currently supports this function
